src/views/layout.njk

```diff
@@ -86,6 +79,7 @@
   }) }}
 {% endblock %}
 
+
```

Comment:
I think we should remove that line, change itself bring no value and just makes diff bigger.